components/MessageReplySection.vue

```diff
@@ -220,7 +220,7 @@ export default {
           // Show the new user modal.
           this.newUserPassword = ret.password
           this.showNewUser = true
-          await this.waitForRef('newUserModal')
+
           // Now that we are logged in, we can reply.
```

Comment:
Again, I think we need a `nextTick` to be sure that the ref will have been added to the DOM.  If it isn't, then we would silently fail to show the modal.  

Or better still, we could move this modal into a separate component and use the standard way of showing it.

Reply:
I'm adding the awaited `nextTick` for now, but will create a ticket for Ilia to rewrite the login (and related) modals to use the updated way of controlling them. Remember it has been skipped in the PR where we addressed all the other ones?

Reply:
Yes, I remember.  That's fine.